.gitignore

```diff
@@ -51,3 +51,4 @@ src/*.js.map
 
 # Exclude the bundle demo file
 !docs/bundle.js
+package-lock.json
```

Comment:
We shouldn't ignore [`package-lock.json`](https://docs.npmjs.com/files/package-locks#using-locked-packages), so everyone can get the exact same dependency tree. This repository uses `yarn`. Ideally we should use only yarn and have both `yarn.lock` and `package-lock.json` files.

Reply:
aah haven't notice that. dropped that commit